docs/conditions.md

```diff
@@ -6,50 +6,54 @@ weight: 11
 
-- [Syntax](#syntax)
-  - [Check](#check)
-  - [Parameters](#parameters)
-  - [Resources](#resources)
-- [Labels and Annotations](#labels-and-annotations)
-- [Examples](#examples)
+## Overview
 
-## Syntax
+A `Condition` resource in Tekton allows you to conditionalize the execution of `Tasks` within a `Pipeline`.
+You define each `Condition` within your `PipelineRun` definition and then conditionalize each desired `Task`
+in the corresponding `Pipeline` definition. 
 
-To define a configuration file for a `Condition` resource, you can specify the
-following fields:
+The `Condition` resource runs its own container image that executes the logic that evaluates your chosen condition.
+This container runs to completion and must return an exit code value of `0` fo the `check` to be successful; all
+other exit codes result in a failure.
+
+**Note:** [Labels](labels.md) and annotations specified in the `Condition's` metadata are automatically
+propagated to the `Pod`.
+
+## Configuring a `Condition`
+
+A `Condition` definition supports the following fields:
 
 - Required:
   - [`apiVersion`][kubernetes-overview] - Specifies the API version, for example
-    `tekton.dev/v1alpha1`.
-  - [`kind`][kubernetes-overview] - Specify the `Condition` resource object.
-  - [`metadata`][kubernetes-overview] - Specifies data to uniquely identify the
-    `Condition` resource object, for example a `name`.
+    `tekton.dev/v1beta1`.
+  - [`kind`][kubernetes-overview] - Identifies this resource object as a `Condition` object.
+  - [`metadata`][kubernetes-overview] - Specifies metadata that uniquely identifies this
+    `Condition` object. For example, a `name`.
   - [`spec`][kubernetes-overview] - Specifies the configuration information for
-    your `Condition` resource object. In order for a `Condition` to do anything,
-    the spec must include:
-    - [`check`](#check) - Specifies a container that you want to run for evaluating the condition
-    - [`description`](#description) - Description of the Condition.
+    this `Condition` resource object. This must include:
+    - [`check`](#check) - Specifies a container that you want to run for evaluating this `Condition`.
+    - [`description`](#description) - Provides a meaningful description of this `Condition` object.
 
 [kubernetes-overview]:
   https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#required-fields
 
-### Check
-
-The `check` field is required. You define a single check to define the body of a `Condition`. The 
-check must specify a [`Step`](./tasks.md#steps). The container image runs till completion. The container 
-must exit successfully i.e. with an exit code 0 for the condition evaluation to be successful. All other 
-exit codes are considered to be a condition check failure.
+### Specifying the condition `check` 
 
-### Description
+The `check` field (required) specifies a single piece of evaluation logic that you want to run before the
```

Comment:
how about having `required` in bold?

Reply:
I think bold would negate the parentheses - this is a piece of additional or "by-the-way" information while we explain what the field does.